hugo/content/documentation/user/mumble-url/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Mumble URL"
+---
+Mumble defines it's own `mumble://` URL scheme to share links to servers and channels and to be able to connect to them directly.
+
+On Windows the installer will register `mumble://` links to be opened with Mumble.
+
+Mumble links can be opened on the command line by passing them to mumble.exe too:
+
+```
+mumble.exe mumble://example.com/?version=1.2.0
+```
+
+or with a channel path:
+
+```
+mumble.exe mumble://example.com/ChanA/ChanB?version=1.2.0
+```
+
+You can get get a URL in the Channel context menu (right click):
+
+![Screenshot of Mumble channel context menu Copy URL item](screenshot-chan-copy-url.jpg)
+
+## Connect Behavior
+
+When opening a Mumble link the Mumble client will connect to the server.
+
+* If you are already registered you will connect with your account
+* If the URL contains a username that username will be used
+* If no username is determined, the client will ask with what name you want to connect with
+
+## URL Format
+
+```
+mumble://[username[:password]@]<address>[:port]/[channelpath][?version=<serverversion>][&title=<servername>][&url=<serverurl>]
+```
+
+* optional `username` and `password`
+* mandatory `address` is mandatory; a hostname, IPv4, or IPv6 address
+* optional `port`; defaults to 64738
+* optional `channelpath`; specifies a channel subtree to join; if the Mumble client can join the server but not the channel it will stay in the default channel

```

Comment:
```suggestion
* optional `channelpath`; specifies a channel subtree to join; special characters must be URL encoded; if the Mumble client can join the server but not the channel it will stay in the default channel

```

Reply:
to make it explicit: copying an url does use url-encoded characters

I would expect slashes in channel names to be url-encoded too

the client would be able to handle it correctly then; I did not test it though